File: output/CodeLlama/codenet/Java/Python/s393629232.py
import sys

n = int(sys.stdin.readline())
maxv = -sys.maxsize
minv = int(sys.stdin.readline())
for i in range(1, n):
    ri = int(sys.stdin.readline())
    maxv = max(maxv, ri - minv)
    minv = min(minv, ri)
print(maxv)

File: output/ChatGPT/codenet/Java/C++/s393629232.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    int maxv = INT_MIN;
    int minv;
    cin >> minv;
    for (int i = 1; i < n; i++) {
        int ri;
        cin >> ri;
        maxv = max(maxv, ri - minv);
        minv = min(minv, ri);
    }
    cout << maxv << endl;
    return 0;
}